indexer/Cargo.toml

```diff
@@ -34,21 +34,19 @@ tracing = "0.1"
 tracing-subscriber = { version = "0.2", features = ["env-filter"] }
 wasmer = "2.0"
 wasmer-compiler-cranelift = { version = "2.0" }
-wasmer-compiler-llvm = { version = "2.0", optional = true }
 wasmer-engine-universal = "2.0"
 
 [dev-dependencies]
 chrono = { version = "0.4", features = ["serde"] }
-fuel-core = { version = "0.2", features = ["test-helpers"] }
-fuel-gql-client = { version = "0.2", features = ["test-helpers"] }
+fuel-core = { version = "0.6.4", features = ["test-helpers"] }
+fuel-gql-client = { version = "0.6.4", features = ["test-helpers"] }
 fuel-indexer = { path = "../lib" }
 fuel-storage = { version = "0.1" }
-fuel-types = { version = "0.3", features = ["serde-types"] }
-fuels-abigen-macro = { version = "0.2" }
-fuels-rs = { version = "0.2" }
+fuel-types = "0.3"
+fuels = "0.12"
+fuels-abigen-macro = { version = "0.12" }
```

Comment:
Looks like `fuels` and `fuels-abigen-macro` are on `0.12` here whereas other crates in this workspace are pointing at `0.15` - should we update these too?

It might be worth double-checking the `Cargo.lock` for any dupes that can be resolved. Normally you can spot these by checking the `dependencies` lists within `Cargo.lock`, as duplicate crates need to have their semver version appended for disambiguation.